### PR TITLE
Getting started with Neovim's built-in LSP

### LINKED ISSUE #67: Add neovim setup instructions using the built-in LSP
Thanks for a great project :) The built-in LSP in neovim was released about 6 months ago, in version 0.5.0. The current version on Homebrew is 0.6.1. What do you think about adding setup instructions for it to the Getting Started page?

I'd be happy to start working on it.

### PR DESCRIPTION
**Pull request by slarwise**
Fixes #67

### CODE REVIEW

File: docs/getting-started/vim.md [hunk 1]
```diff
@@ -55,3 +50,53 @@ message from Coc:
...
+
+Install using, for example, [vim-plug](https://github.com/junegunn/vim-plug):
+
+```vim
+Plug 'neovim/nvim-lspconfig'
+```
+
+### Enable the language server
+
+Add the following setup call to your init.vim to make the language server attach
```
Comment (robertoaloi) [inline docs/getting-started/vim.md:71]:
```suggestion
Add the following setup call to your `init.vim` to make the language server attach
```

File: docs/getting-started/vim.md [hunk 2]
```diff
@@ -55,3 +50,53 @@ message from Coc:
...
+to Erlang files:
+
+```vim
+lua << EOF
```
Comment (robertoaloi) [inline docs/getting-started/vim.md:75]: Is this really supposed to say `lua`? :)

File: docs/getting-started/vim.md [hunk 3]
```diff
@@ -55,3 +50,53 @@ message from Coc:
...
+Run `:LspInfo` to get the status of active and configured language servers.
+
+### Configuration
+
+There are no default keybindings for the LSP commands. You can find a good
+example on how to set these up in the
+[Keybindings and completion](https://github.com/neovim/nvim-lspconfig#keybindings-and-completion)
+section on the nvim-lspconfig plugin page. The idea is to create a function that
+sets up your keybindings and configuration, which is passed to the setup
+function in the previous section. This function is then called when the erlang
```
Comment (robertoaloi) [inline docs/getting-started/vim.md:94]:
```suggestion
function in the previous section. This function is then called when the Erlang
```

**Review by robertoaloi — COMMENTED** [review]
Nice stuff, thanks for putting this guide together! I left just a few minor comments.

File: docs/getting-started/vim.md (hunk 2 above)
Comment (slarwise) [reply docs/getting-started/vim.md]: Oops, no it is not :)

**Comment by slarwise** [comment]
Thanks for reviewing, I have fixed your comments :) Let me know if I should squash the commits or anything else

File: docs/getting-started/vim.md [hunk 4]
```diff
@@ -55,3 +50,51 @@ message from Coc:
...
+Plug 'neovim/nvim-lspconfig'
+```
+
+### Enable the language server
+
+Add the following setup call to your `init.vim` to make the language server attach
+to Erlang files:
+
+```vim
+lua require'lspconfig'.erlangls.setup{}
```
Comment (robertoaloi) [inline docs/getting-started/vim.md:75]: One more `lua` remain?

Comment (robertoaloi) [reply docs/getting-started/vim.md:75]: TIL

**Pull request merged**